source/general/faq.rst

```diff
@@ -32,3 +32,43 @@ We encourage members to make suggestions! Suggestions do not need to be based of
 you had seen in the show, but they should remain practical. You can make suggestions by
 clicking `here to go to our forums <https://projectkorra.com/forum/forums/suggestions.8/>`_.
 Please ensure that your suggestion is not a duplicate of another already posted!
+
+Where can I download ProjectKorra?
+==================================
+
+The latest release of ProjectKorra is 1.8.9, which works for Minecraft 1.15 and 1.14. You can download it `here <https://projectkorra.com/forum/resources/projectkorra-core.1/>`_.
+
+Does ProjectKorra work for 1.16?
+================================
+
+The latest release, 1.8.9, was designed for Minecraft 1.15 and 1.14.
+If you installed it on your 1.16 server, the plugin would mostly work, but it would break under certain circumstances. Therefore, it's not recommended to use PK 1.8.9 for Minecraft 1.16.
```

Comment:
`If you installed it on an incompatible version, the plugin would not be guaranteed to work. Using ProjectKorra with versions other than the one(s) listed above is not recommended.`